### PR TITLE
Make test_push_composer_start_more reliable

### LINKED ISSUE #3698: CI failings, probably celery worker isn't started
I did some investigation on the CI failings, it seems that Celery worker isn't started. I think that `/etc/bodhi/start.sh` is not executed when docker image changes from `f31-integration-build-bodhi` to `f31-integration`
https://github.com/fedora-infra/bodhi/blob/6794e5b9b5b2ecca77e43e7df596e8b569b84448/devel/ci/integration/bodhi/Dockerfile-f31#L65
I tried to change that from CMD to ENTRYPOINT, but it never executes - I cannot even see a simple echo command.

That leads the 
https://github.com/fedora-infra/bodhi/blob/6794e5b9b5b2ecca77e43e7df596e8b569b84448/devel/ci/integration/tests/test_bodhi_tasks.py#L48
line fail for timeout.
Note that the other tasks that rely on celery `.delay()` are not tested for a successful completion so the other tests don't fail...

### PR DESCRIPTION
**Pull request by AdamSaleh**
The test relies on having pending composes that will run pungi

We only check there are composes, we should be checking that their type
implies running pungi as well, so either module or rpm content_type.
I hope adding more thorough query should help with the test.

The new query:

```
select distinct
        c.release_id, c.request, state, b.type
      from
        composes as c
      inner join updates as u on (c.release_id = u.release_id and c.request = u.request )
      inner join builds as b on (b.update_id = u.id)
      where b.type in ('rpm', 'module') and state <> 'failed';
```

On failure, it will print the result of the query to test-log.

Fixes #3698

 Maybe :smile:

### CODE REVIEW

File: devel/ci/integration/tests/test_bodhi_tasks.py [hunk 1]
```diff
@@ -35,17 +35,34 @@ def test_push_composer_start(bodhi_container, db_container, rabbitmq_container):
     output = "".join(line.decode("utf-8") for line in output)
     assert output.endswith("Requesting a compose\n")
     # Check how many composes should be done
-    query = "SELECT COUNT(*) FROM composes"
+    # because we later check for pungi.log, we need to see if there are composes that apply
+    # we can get these from the type field of the builds table,
+    # after joining with the composes table through updates table.
+    query = """
```
Comment (Zlopez) [inline devel/ci/integration/tests/test_bodhi_tasks.py:41]: Could you write the SQL keywords in uppercase, so it is more readable?

File: devel/ci/integration/tests/test_bodhi_tasks.py [hunk 2]
```diff
@@ -35,17 +35,34 @@ def test_push_composer_start(bodhi_container, db_container, rabbitmq_container):
...
+        c.release_id, c.request, state, b.type
+      from
+        composes as c
+      inner join updates as u on (c.release_id = u.release_id and c.request = u.request )
+      inner join builds as b on (b.update_id = u.id)
+      where b.type in ('rpm', 'module') and state <> 'failed';
+    """
     db_ip = db_container.get_IPv4s()[0]
     conn = psycopg2.connect("dbname=bodhi2 user=postgres host={}".format(db_ip))
+    print("dbname=bodhi2 user=postgres host={}".format(db_ip))
```
Comment (Zlopez) [inline devel/ci/integration/tests/test_bodhi_tasks.py]: This is debug print or do you have this here on purpose?

**Review by Zlopez — COMMENTED** [review]
Few comments

File: devel/ci/integration/tests/test_bodhi_tasks.py (hunk 2 above)
Comment (AdamSaleh) [reply devel/ci/integration/tests/test_bodhi_tasks.py]: Yes, this is a left-over debug print.

File: devel/ci/integration/tests/test_bodhi_tasks.py (hunk 1 above)
Comment (AdamSaleh) [reply devel/ci/integration/tests/test_bodhi_tasks.py:41]: I am actually used to writing sql lower-case, but I can change it :)

Comment (Zlopez) [reply devel/ci/integration/tests/test_bodhi_tasks.py:41]: It's usually more readable uppercase, especially inside python string.

**Review by Zlopez — APPROVED** [review]
:+1:

**Pull request merged**